app/src/vuex/modules/session.js

```diff
@@ -60,6 +60,7 @@ export default ({ apollo }) => {
     },
     setExperimentalMode(state) {
       state.experimentalMode = true
+      state.developmentMode = true
```

Comment:
?

Reply:
why are doing 2 modes?

Reply:
otherwise I'd need to add a "develoment" flag so this kind of things can be tested on a preview

Reply:
But I can do that. What you prefer

Reply:
I switched it to just display in experimental mode